Sources/CLI/Volume/VolumeList.swift

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import ContainerClient
+
+extension Application.VolumeCommand {
+    struct VolumeList: AsyncParsableCommand {
+        static let configuration = CommandConfiguration(
+            commandName: "list",
+            abstract: "List volumes",
+            aliases: ["ls"]
+        )
+
+        @Flag(name: .shortAndLong, help: "Don't truncate output")
+        var noTrunc: Bool = false
```

Comment:
Is this just if some of the headers go off the screen, don't replace them with `...`?

Reply:
@dcantah I know with `container image ls` the image digest is always elided with `...` at 24 characters. This actually seems a good idea but it might be good to indicate what would be truncated here (`Don't truncate {} output`), and also apply it uniformly wherever else we're dropping information (if it's only image, I wouldn't be against revising this PR with that change).

Reply:
Right, but all the info for the volume is user provided, so is it just if the user supplied a name like "my-really-cool-volume-name-that-is-probably-a-bit-too-unique"

Reply:
Right now it's the responsibility of `list` subcommand to trim any fields it wants to (`image list` with the manifest digest is the only place I'm aware of today). Perhaps we just drop the option and output the name without trimming.